main.go

```diff
@@ -365,7 +394,27 @@ func runCLIApp(c *cli.Context) (err error) {
 			err = fmt.Errorf("daemonize.Run: %w", err)
 			return
 		} else {
-			logger.Infof(SuccessfulMountMessage)
+			if isDynamicMount {
+				logger.Infof(SuccessfulMountMessage)
+			} else {
+				switch flags.MetadataPrefetchMode {
+				case config.MetadataPrefetchModeDisabled:
+					logger.Infof(SuccessfulMountMessage)
```

Comment:
The successfulMountMessage is being logged in all cases. Better to log it at the end which should save you three lines of code.

Reply:
> The successfulMountMessage is being logged in all cases. 

Yes, but always in the same order. In async case, the log is before running callListRecursive, but it is after the callListRecursive in sync case. So, I don't see any easy way around this.

> Better to log it at the end which should save you three lines of code.

That'll change output to in case of async.